builtin/credential/cert/path_crls.go

```diff
@@ -29,6 +33,10 @@ May be DER or PEM encoded. Note: the expiration time
 is ignored; if the CRL is no longer valid, delete it
```

Comment:
The previous comment is incorrect right? It is `The public CRL that should be trusted to attest to cert's validity statuses` or something similar?

Reply:
Yep.